rpc/sigpipe.go

```diff
@@ -0,0 +1,9 @@
+// +build !darwin
+
+package rpc
+
+import "net"
+
+func DisableSigPipe(c net.Conn) error {
+	return nil
```

Comment:
Sure, how about something like:

`return errors.New("DisableSigPipe not supported")`